sesame/src/main/scala/org/w3/banana/sesame/SesameHttpSparqlEngine.scala

```diff
@@ -0,0 +1,57 @@
+package org.w3.banana.sesame
```

Comment:
Although not the same as the sparql implementation, if you look at [SesameGraphSparqlEngine]( https://github.com/banana-rdf/banana-rdf/blob/series/0.8.x/sesame/src/main/scala/org/w3/banana/sesame/SesameGraphSparqlEngine.scala) that "looks" more like the jena implementation and what is (ideally) need here. (in fact.... almost the same.....)